setup.py

```diff
@@ -33,7 +33,7 @@
         # License: BSD
         # Upstream url: https://github.com/pyca/cryptography
         # Use: For encryption
-        'cryptography>=1.2.1,<3.0.0',
+        'cryptography>=1.2.1,<4.0.0',
```

Comment:
Let's just remove the upper-bound, since it seems crpytography changed their version scheme and will publish more major releases pretty frequently

Reply:
Update as remove upper bound